pyproject.toml

```diff
@@ -36,6 +36,8 @@ pytest-bdd = "^4.0.1"
 cryptography = "^3.2.1"
 python-dateutil = "^2.8.1"
 pytest-rerunfailures = "^9.1.1"
+api-test-utils = {url = "https://github.com/NHSDigital/apim-test-utils/releases/download/v1.1.8-alpha/api_test_utils-1.1.8a0-py3-none-any.whl"}
```

Comment:
Why did we need to re-version the test utils?

Reply:
Had trouble with sim auth. tried changing utils version, but wasn't related. I'll revert now.